conf/dpvs.conf.single-nic.sample

```diff
@@ -129,6 +129,7 @@ worker_defs {
     <init> worker   cpu8 {
         type        slave
         cpu_id      8
+        icmp_redirect_core
```

Comment:
Please add the config into `conf/dpvs.conf.items`, which should be a full config list.

Reply:
Add the config into conf/dpvs.conf.items.